web/packages/core/src/ruffle-player.ts

```diff
@@ -1307,7 +1307,7 @@ export function isScriptAccessAllowed(
  * @returns True if the built-in context items should be shown.
  */
 export function isBuiltInContextMenuVisible(menu: string | null): boolean {
-    if (menu === "true" || menu === null) {
+    if (menu.toLowerCase() === "true" || menu === null) {
```

Comment:
should probably test those in the opposite order -- won't `menu.toLowerCase()` error if menu is null?

Reply:
Yeah, quick fixes are not always good fixes. One moment, I'll fix this.